blocks.config.json

```diff
@@ -2,59 +2,53 @@
   {
     "type": "file",
     "id": "code-block",
-    "title": "Code block",
-    "description": "A basic code block",
-    "sandbox": false,
+    "title": "Code",
+    "description": "Read & edit code",
     "entry": "blocks/file-blocks/code/index.tsx",
     "matches": ["*"],
     "example_path": "https://github.com/githubnext/blocks-tutorial/blob/main/Aside.jsx"
   },
   {
     "type": "file",
-    "id": "excalidraw-block",
-    "title": "Drawing block",
-    "description": "A whiteboard tool",
-    "sandbox": false,
+    "id": "excalidraw",
```

Comment:
maybe we should leave the ids alone? they don't matter much in the UI, and there is metadata that references them that would need updating.

Reply:
I think we're okay for these really specific use cases - Blocks will default to the next most specific block if it can't find the one specified, so things like https://blocks.githubnext.com/githubnext/blocks/blob/main/examples/drawing.excalidraw?blockKey=githubnext__blocks-examples__excalidraw2&fileRef= will still work. More than gappy to revert if you feel strongly, though!

Reply:
er, happy, not gappy

Reply:
do not feel strongly, garry on